### ISSUE TITLE
data_layout isn't used at batch_norm_layer

### ISSUE THREAD

**Issue by tonyyang-svail** [PaddlePaddle/Paddle#9803]
https://github.com/PaddlePaddle/Paddle/blob/add367c3f4b77103cda50b9359492f34a366477c/python/paddle/fluid/layers/nn.py#L1508-L1519

batch norm layer in nn.py has an input layer_out, but it isn't used in the op attribute 

https://github.com/PaddlePaddle/Paddle/blob/add367c3f4b77103cda50b9359492f34a366477c/python/paddle/fluid/layers/nn.py#L1574-L1592
